api/contract/cli.yml

```diff
@@ -12,7 +12,7 @@ paths:
   /setup:
     $ref: "./openapi/src/common/paths/setup.yml"
   /write:
-    $ref: "./openapi/src/common/paths/write.yml"
```

Comment:
It can. `common/paths/write.yml` got moved to `oss/paths/write.yml`, and `cloud/paths/write.yml` was created to handle different sets of potential error codes between the two. We eventually need to reconcile the two but I didn't want to do that work as part of this.